bin/package/sudoers/mysterium-node

```diff
@@ -5,3 +5,4 @@ mysterium-node    ALL=NOPASSWD: /sbin/iptables
 mysterium-node    ALL=NOPASSWD: /sbin/ip
 mysterium-node    ALL=NOPASSWD: /etc/mysterium-node/prepare-env.sh
 mysterium-node    ALL=NOPASSWD: /sbin/tc
+mysterium-node    ALL=NOPASSWD: /sbin/modeprobe
```

Comment:
shouldn't this be `modprobe` @soffokl ?

Reply:
Thanks, it should be `modprobe`, I'll fix it.